lib/stock_markets/file_data_processor.rb

```diff
@@ -19,13 +21,32 @@ class FileDataProcessor
     # @return [StockMarkets::FileDataProcessor] instance of object for file data processing
     def initialize
       @data = []
-      load_from_disk!
+      @parsed_csv = load_from_network || load_from_disk!
```

Comment:
on each initialize, we will try to load data from the network. I would recommend doing a separate task (it could be rake) to update the local file. I don't think, that it is good to pull data from the network each time